retries.go

```diff
@@ -0,0 +1,68 @@
+package linodego
+
+import (
+	"log"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/go-resty/resty/v2"
+)
+
+// type RetryConditional func(r *resty.Response) (shouldRetry bool)
+type RetryConditional resty.RetryConditionFunc
+
+var retryConditionals []RetryConditional
+
+// Configures resty to
+// lock until enough time has passed to retry the request as determined by the Retry-After response header.
+// If the Retry-After header is not set, we fall back to value of SetPollDelay.
+func configureRestyRetries(resty *resty.Client) {
+	resty.
+		SetRetryCount(1000).
```

Comment:
I think to default to these loose constraints is fine for now. We should probably circle back and provide a way for consumers to customize the calls to `SetRetryCount` and `SetretryMaxWaitTime`, and merge with defaults.

Reply:
I just reread your description and you covered this. My bad!